.github/workflows/build_test.yml

```diff
@@ -31,6 +31,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
+        pkg_mgr: ['apt', 'conda']
         stage: [base_python, moab, dagmc, openmc]
         hdf5: ['']
         include:
```

Comment:
Yes, because the image names now contain the name of the package manager that was used: `ghcr.io/pyne/pyne_ubuntu_22.04_py3_${{ matrix.pkg_mgr }}${{ matrix.hdf5 }}/${{ matrix.stage }}:stable`. This image name change was also made in `docker_publish.yml` so that we aren't writing to the same image name when building with `apt` vs building with `conda`.

Reply:
Sorry - I wasn't clear - I think these special includes still need to be updated to include the package manager choice, don't they?

Reply:
Yep, good catch! I checked the `build_test.yml` [workflow run](https://github.com/bquan0/pyne/actions/runs/6960830453) and it only had a `dagmc, hdf5` run instead of two runs, one with `apt` and another with `conda`. I'll make those changes.